Sources/FingerprintJS/Harvesters/DataExchange/DeviceInfo.swift

```diff
@@ -1,16 +1,10 @@
-//
-//  DeviceInfo.swift
-//  FingerprintJS
-//
-//  Created by Petr Palata on 15.04.2022.
-//
-
-import CoreGraphics
 import Foundation
 
 public struct DeviceInfo {
     public let vendorIdentifier: UUID?
 
+    public let userInterfaceStyle: UserInterfaceStyle
```

Comment:
This property should have some kind of public documentation. I know the rest of the struct doesn't but that's an oversight.

Reply:
Added.